### ISSUE TITLE
`gh label list` Fails Within Repository

### ISSUE THREAD

**Issue by Jordan-Dennis** [heaths/gh-label#27]
Hi @heaths,
I've recently downloaded your extension and when I tried to list the labels I received the following error,
```
error: cannot find gh; is it installed? error: exec: "gh": executable file not found in $PATH
```
However, I am able to use `gh` normally and `gh label --help` works. Here is my path which contains `gh` twice (exported again to try and pix this issue)
```
/usr/bin/latexmk:/snap/bin/gh:/usr/bin/latexmk:/home/jordan/anaconda3/bin:/home/jordan/anaconda3/condabin:/home/jordan/.opam/default/bin:/home/jordan/.cargo/bin:/home/jordan/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:~/Documents/flutter/bin:~/Documents/flutter/bin:/snap/bin/gh
```
Regards
Jordam

**Comment by heaths** [comment]
How did you download it? Did you run `gh extension install heaths/gh-label`? That's the only method supported for downloading `gh` extensions.

Note, however, this really isn't needed anymore. There's a built-in `gh label` and I added what functionality this did have it didn't to the built-in `gh label` several versions ago.

**Comment by Jordan-Dennis** [comment]
Yes I did use `gh extension install heaths/gh-label`. I expect my `gh` is out of date is all. I'll try updating it. :+1:
